sklearn/feature_selection/__init__.py

```diff
@@ -16,6 +16,7 @@
 from .univariate_selection import GenericUnivariateSelect
 
 from .variance_threshold import VarianceThreshold
+from .correlation_threshold import CorrelationThreshold
```

Comment:
in the interest of reducing confusion about "public" and "private" api, I think we've recently been doing `_filename`, haven't we?